ui/raidboss/data/06-ew/raid/p10n.txt

```diff
@@ -1,6 +1,81 @@
 ### P10N: Anabaseios: The Tenth Circle
+#
+# -ii 825E 825F 8261 8263 8277 8279 825B 825C 8266 8269 8271 8273 8274 826B 826C 826D 826E 86B5 86B6 86B7 86B8
+#
+# Ignored Ids:
+# 825[EF] - 'Bury' action/damage from Pandaemoniac Pillars
+# 8261      'Imprisonment' extra
+# 8263      'Cannonspawn' extra
+# 8277      'Pandaemonic Meldown' extra
+# 8279      'Soul Grasp' extra
+# 825[BC]   'Entangling Web' extra
+# 8266      'Pandaemonic Ray' extra
+# 8269      'Touchdown' extra
+# 8271      'Parted Plumes' blades
+# 827[34]   'Wicked Step' individual towers
+# 826[B-E]  Supplemental AoE damage from 'Harrowing Hell'
+# 86B[5-8]  Supplemental AoE damage from 'Harrowing Hell'
+
+# Note: The boss's name has a Unicode character (�)  which is causing syncs to fail.
+# The character is manually escaped in the syncs with the correct codepoint (U+00E6).
+# Entry names use "ae" as separate characters to ensure proper UI display.
 
 hideall "--Reset--"
 hideall "--sync--"
 
 0.0 "--Reset--" sync / 21:........:4000000F:/ window 100000 jump 0
+
+0.0 "--sync--" sync /Engage!/ window 0,1
+2.5 "--sync--" sync / 1[56]:[^:]*:Pand\u00e6monium:827A:/
+10.9 "Silkspit (marked)" sync / 1[56]:[^:]*:Pand\u00e6monium:8259:/
```

Comment:
If this has a cast, can you add in a `14` line sync with a larger window (e.g. `golbez-ex.txt`) just in case the first auto is inconsistent?